src/ResultPage.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+export default class ResultPage extends React.Component {
+  displayResult(displayPage, student) {
+    this.props.displayResult(displayPage, student);
+  }
+
+  render() {
+    return (
+      <div>
+        Click on the person to get the detailed result.
+        <ul><b>FirstName LastName Percentage </b></ul>
+        {
+          this.props.studentsData.map((student, i) => {
+            let total = (student.marks.english + student.marks.hindi + student.marks.mathematics);
```

Comment:
use only one let 
use object destructuring

Reply:
resolved in eslint correction.